src/arbitrum/instantiate_bridge.ts

```diff
@@ -3,6 +3,14 @@ import { providers } from 'ethers'
 
 const networkID = 42161
 
+const l2Rpc = (chainId: number) => {
+  if (chainId === 42161) return 'https://arb1.arbitrum.io/rpc'
+  else if (chainId === 421611) return 'https://rinkeby.arbitrum.io/rpc'
+  else if (chainId === 42170) return 'https://nova.arbitrum.io/rpc'
+  else if (chainId === 421613) return 'https://goerli-rollup.arbitrum.io/rpc'
+  throw new Error('No L2 RPC detected')
+}
+
 // ref: https://github.com/OffchainLabs/arb-token-lists/blob/master/src/lib/instantiate_bridge.ts
```

Comment:
see the current version of the file at this URL to understand why the changes in this file are needed